bin/update_go

```diff
@@ -38,7 +38,7 @@ get_version() {
     for i in "Dockerfile" "Dockerfile.amd64"; do
         if [ -e  "${repo_path}/${i}" ]; then
              info ${repo_path}
```

Comment:
nit: shellcheck suggests to double quote variables info "${repo_path}"

Reply:
Presently the CI is exclusively checking for warnings, and this flagged error is classified under the log level info.
I'll add support for checking the other log levels and address the necessary changes on line 40 in  another pr shortly.
Thanks @crosas=3p!

Reply:
See #42 for resolution.